extension/icu/CMakeLists.txt

```diff
@@ -32,9 +32,9 @@ link_threads(icu_extension)
 if(NOT WITH_INTERNAL_ICU)
   find_package(
     ICU
-    COMPONENTS i18n
+    COMPONENTS i18n uc
     REQUIRED)
-  target_link_libraries(icu_extension ICU::i18n)
+  target_link_libraries(icu_extension ICU::i18n ICU::uc)
```

Comment:
It looks like this was missed because we forgot to set `WITH_INTERNAL_ICU=OFF` :)